README.md

```diff
@@ -47,24 +47,36 @@ React Native boilerplate with mostly used features.
 
 - `yarn lint` Runs eslint checks;
 - `yarn ts-compile-check` Runs TS compiling checks;
+- `yarn rename "New App Title" com.new.appname` Rename application and change package/bundle ids;
 
 ## Getting Started
 
-Environment requirements:
+0. Environment requirements:
+    - NodeJS
```

Comment:
Maybe need to write a min nodejs version?

Reply:
Yep (: